doc/users/whats_new/2015-05_interactive_OO.rst

```diff
@@ -0,0 +1,37 @@
+Interactive OO usage
+--------------------
+
+All `Artists` now keep track of if their internal state has been
+changed but not reflected in the display ('stale') by a call to
+``draw``.  It is thus possible to pragmatically determine if a given
+`Figure` needs to be re-drawn in an interactive session.
+
+To facilitate interactive usage a ``draw_all`` method has been added
+to ``pyplot`` which will redraw all of the figures which are 'stale'.
+
+To make this convenient for interactive use matplotlib now registers
+a function either with IPython's 'post_execute' event or with the
+displayhook in the standard python REPL to automatically call
+``plt.draw_all`` just before control is returned to the REPL.  This ensures
+that the draw command is deferred and only called once.
+
+The upshot of this is that for interactive backends (including notebook)
```

Comment:
I think we should make it a bit more explicit to make sure that users aren't confused between the notebook with inline backend and the notebook backend. I would write something like "(including notebook i.e. '%matplotlib notebook') "